client/status/common.go

```diff
@@ -45,7 +45,7 @@ func (x ServerInternal) Error() string {
 func (x ServerInternal) Is(target error) bool {
 	switch target.(type) {
 	default:
-		return false
+		return errors.Is(ErrAPI, target)
```

Comment:
can you, please, clarify a little how that change works? cause i failed a little here

Reply:
It's a dirty hack to check error for multiple types in go < 1.20

Reply:
BTW, have you tried `errors.Is(errors.New("garbage"), apistatus.ErrServerInternal`? Can we have a test for it?